Source/Tools/SpritePacker/SpritePacker.cpp

```diff
@@ -328,7 +328,8 @@ void Run(Vector<String>& arguments)
     spriteSheetImage.SetSize(packedWidth, packedHeight, 4);
 
     // zero out image
-    spriteSheetImage.SetData((unsigned char*)calloc(sizeof(unsigned char), (size_t)packedWidth * packedHeight * 4));
+    // Ugly: depends on implementation details - is this really needed?
+    memset(spriteSheetImage.GetData(), 0, packedWidth * packedHeight * 4);
```

Comment:
Incidentally I have just fixed this leak yesterday in the `clang-tidy-up` dev branch. So, you could remove this fix in your PR.